src/entity/EvalutionQuestion.ts

```diff
@@ -1,11 +1,21 @@
-import { PrimaryGeneratedColumn, Column, Entity, ManyToOne } from 'typeorm';
+import {
+  PrimaryGeneratedColumn,
+  Column,
+  Entity,
+  ManyToOne,
+  Unique,
+} from 'typeorm';
 import { Pool } from '@/entity/Pool';
 
 @Entity()
+@Unique(['pool', 'questionIndex'])
```

Comment:
I don't believe this will work as expected. You should explicitly add `poolId` as a column and use that for the unique constraint

Reply:
Makes sense. Updating